dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/ClouderaCMFHostsTask.java

```diff
@@ -60,6 +67,8 @@ protected Void doRun(
 
     final URI baseURI = handle.getBaseURI();
     try (Writer writer = sink.asCharSink(StandardCharsets.UTF_8).openBufferedStream()) {
+      ApiHostListDto apiHosts;
```

Comment:
Variables should be defined close to usage place for better readability and code structure. In the current case we can add items to `hosts` under `try` statement.

Reply:
Fixed